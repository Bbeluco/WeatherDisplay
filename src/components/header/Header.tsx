import React from "react";
import style from './Header.module.css'

function Header(): React.JSX.Element {
    return (
        <>
            <h1 className={style.headerTitle}>Previsao do tempo</h1>
        </>
    )
}

export default Header;