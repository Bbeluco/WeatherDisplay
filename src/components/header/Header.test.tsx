import Header from "./Header"
import { render, screen } from "@testing-library/react"
import '@testing-library/jest-dom'
import TestRenderer from 'react-test-renderer';


describe('< Header/>', () => {
    it('should render with correct text', () => {
        render(<Header />)
        expect(screen.getByText('Previsao do tempo')).toBeInTheDocument()
    })

    it('should test if render properly', () => {
        const component = TestRenderer.create(<Header />).toJSON()

        expect(component).toMatchSnapshot()
    })
})


export default {}