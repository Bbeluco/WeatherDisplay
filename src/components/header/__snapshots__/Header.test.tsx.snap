// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`< Header/> should test if render properly 1`] = `
<h1
  className="headerTitle"
>
  Previsao do tempo
</h1>
`;
